lib/bundle/brew_dumper.rb

```diff
@@ -17,15 +17,18 @@ def self.formulae
 
     def self.dump
       formulae.map do |f|
-        if f[:args].empty?
-          "brew '#{f[:full_name]}'"
-        else
-          args = f[:args].map { |arg| "'#{arg}'" }.sort.join(", ")
-          "brew '#{f[:full_name]}', args: [#{args}]"
-        end
+        restart = started_service?(f[:full_name])
+        brewline = "brew '#{f[:full_name]}'"
+        args = f[:args].map { |arg| "'#{arg}'" }.sort.join(", ")
+        brewline += ", args: [#{args}]" unless f[:args].empty?
+        brewline += "#{restart}"
```

Comment:
`brewline += ", service_restart: true" unless BrewServices.started?(name)` and then could kill `restart` above and `def self.started_service?(name)` below.